frontend/src/features/projects/common/tabs/ProjectInformationTab.tsx

```diff
@@ -30,8 +31,9 @@ const ProjectInformationTab: React.FunctionComponent<IProjectInformationTabProps
       />
 
       <h3>Notes</h3>
-      <ProjectNotes className="col-md-auto" disabled={true} label="Agency Notes" />
+      <ProjectNotes className="col-md-auto" disabled={true} label="Notes" />
```

Comment:
Previously it was thought that the "Notes" field was disappearing; however, they were just labelled Agency Notes after a project was submitted.